### ISSUE TITLE
Text Classification Using Naive Bayes Classifier

### ISSUE THREAD

**Issue by khushi-purwar** [prathimacode-hub/ML-ProjectKart#424]
Define You:

- [x] LGM-SOC'21 Participant
- [ ] DevIncept Participant 
- [x] Contributor

Problem is :  We have to classify a sentence whether it is a question or a statement.

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like...**

**Describe alternatives you've considered?**

**Approach to be followed (optional):**

**Additional context**

**Comment by khushi-purwar** [comment]
@prathimacode-hub  can I work on this issue under LGMSOC21 ?

**Comment by prathimacode-hub** [comment]
Yes, you can work on this issue. But use atleast 3-4 algorithms to gain the efficiency. Tell me your approach you will be using, task you are taking up and algorithms you would be working on. I can assign the issue then. Let me know of the details. @khushi-purwar

**Comment by harshita9621** [comment]
@prathimacode-hub can I work on this issue under DevIncept ?

**Comment by khushi-purwar** [comment]
> Yes, you can work on this issue. But use atleast 3-4 algorithms to gain the efficiency. Tell me your approach you will be using, task you are taking up and algorithms you would be working on. I can assign the issue then. Let me know of the details. @khushi-purwar

Ok.   I will  use Naive Bayes , Support Vector Machine, K-Nearest Neighbors and Logistic Regression. 
And the approach is first I will do the data preprocessing , then i will use different algorithms to build model

**Comment by prathimacode-hub** [comment]
Great, this issue is assigned to you. You can proceed. Make sure you follow the project structure given. @khushi-purwar 

You can take up a new issue to work on, as it's on FCFS basis. @harshita9621

**Comment by khushi-purwar** [comment]
> Great, this issue is assigned to you. You can proceed. Make sure you follow the project structure given. @khushi-purwar
> 
> You can take up a new issue to work on, as it's on FCFS basis. @harshita9621

Ok, I will do it accordingly. Thanks for assigning me this issue.

**Comment by prathimacode-hub** [comment]
Its been 23 days and no PR form you yet. @khushi-purwar

**Comment by prathimacode-hub** [comment]
Update please. @khushi-purwar

**Comment by khushi-purwar** [comment]
I am getting errors in SVM , Logistic regression algorithm, Only Naive byes is working fine. That's why I am not able to complete this issue.

**Comment by prathimacode-hub** [comment]
> I am getting errors in SVM , Logistic regression algorithm, Only Naive byes is working fine. That's why I am not able to complete this issue.

Ok, what's the error you are getting. Else you can try it up with other algos available. @khushi-purwar

**Comment by prathimacode-hub** [comment]
Are you able to do it or not? Atleast reply to msgs. As last date to merge PR's is tomorrow. If not done by tomorrow, I would have to remove the label. Its been near a month since issue is assigned. @khushi-purwar

**Comment by khushi-purwar** [comment]
No, it is not completed. Sorry for the inconvenience.

**Comment by prathimacode-hub** [comment]
Ok, no issue. Hope you could complete it off in 2 days or so. But label has to been removed as per the instructions of the team. @khushi-purwar

**Comment by khushi-purwar** [comment]
Not able to complete.

**Comment by abhisheks008** [comment]
No worries @khushi-purwar. This issue is open, you can take up this issue later on if you wanna contribute in this. All the best for your future open source journey. Good luck! 👍

**Comment by tanvideshmukh29** [comment]
@prathimacode-hub
I would love to contribute as a participant of hacktoberfest2021.
Can you assign this to me?

**Comment by NikitaEmberi** [comment]
@prathimacode-hub ma'am and @abhisheks008 sir, I would like to implement text classification using NLP where I will be fetching tweets from twitter live and classify them using TextBlob library. In addition to this, I would like to add another feature, wherein if the tweets are labelled as being negative, then I will be checking if the user has suicidal tendency using LinearSVC classifier. Please could you assign this issue to me. Thank you.
